### ISSUE TITLE
xt/aspell.t doc/Language/unicode_entry.pod6

### ISSUE THREAD

**Issue by coke** [perl6/doc#2869]
## The problem

```
$ xt/aspell.t doc/Language/unicode_entry.pod6
```
Fails after b4534dbe. Looks like commands that need to be added to the dictionary file.

**Comment by JJ** [comment]
You mean the rxvt configuration commands?

**Comment by coke** [comment]
I didn't read the file or scope the words, just saw the failing test.

**Comment by JJ** [comment]
OTOH, is it worth the while to spell-check code blocks?

**Comment by coke** [comment]
IMO yes, because there's a lot of items in code that are words, like comments, variable names, etc.

Even if they're not words, having the config "words" whitelisted means there's less chance of someone typo-ing them in other sections of the document.

**Comment by JJ** [comment]
Shouldn't we maybe have some special :lang keyword that just skips the
whole block? In cases such as this, it's really cumbersome to add every one
of them (or even whitelist them).

**Comment by coke** [comment]
I'm not sure rewriting the test to avoid code helps, given the large amount of code we *do* want tested. (Including use of official Perl 6 class names in code, e.g.). I don't think that the overhead on keeping the spelling test passing is that great, even in this particular case.

The only reason I posted this as a ticket was so that the original author could verify the spelling of the comands/params before we codified it.
